packages/sdk/src/sdk/api/albums/types.ts

```diff
@@ -58,7 +65,11 @@ const createAlbumTrackMetadataSchema = () =>
   createUploadTrackMetadataSchema().partial({
     genre: true,
     mood: true,
-    tags: true
+    tags: true,
+    isStreamGated: true,
+    streamConditions: true,
```

Comment:
what does setting fields to `true` here do?

Reply:
This just includes those fields in the partial
https://zod.dev/?id=partial